apps/api/app/utils/notification/channels/__init__.py

```
@@ -0,0 +1,14 @@
+from app.utils.notification.channels.external import ExternalPlatformAdapter
+from app.utils.notification.channels.inapp import InAppChannelAdapter
+from app.utils.notification.channels.telegram import TelegramChannelAdapter
+
+__all__ = [
+    "ChannelAdapter",
+    "SendFn",
+    "ExternalPlatformAdapter",
+    "InAppChannelAdapter",
+    "TelegramChannelAdapter",
+    "DiscordChannelAdapter",
+]
```

Ruff (0.15.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)